lib/config.lua

```diff
@@ -35,7 +35,7 @@ function config.loadConfig(file)
         return nil, "File ".. file .." doesn't exist"
     end
     local conf = {}
-    local valid_params = {'API_URL', 'API_KEY', 'LOG_FILE'}
+    local valid_params = {'API_URL', 'API_KEY', 'LOG_FILE', 'LOG_LEVEL'}
```

Comment:
Do we want to add an explicit default value in the code ?

Reply:
the `getLogLevel` returns `logging.INFO` by defaut, might do the trick ?

Reply:
Yep, but I also added default value for the `LOG_FILE` and `LOG_LEVEL` :+1: